ci/prow/boskos/create_projects.sh

```diff
@@ -22,14 +22,13 @@ readonly BILLING_ACCOUNT=${2:?"Second argument must be the billing account."}
 readonly RESOURCE_FILE="resources.yaml"
 
 if [[ ! -f ${RESOURCE_FILE} || ! -w ${RESOURCE_FILE} ]]; then
-  echo "${RESOURCE_FILE} does not exist or is not writable"
+  echo "${RESOURCE_FILE} does not exist or is not writable" && exit 1
```

Comment:
There's no reason to make this a single line, hurting readability. Please split.

Reply:
Done.